pennylane/templates/subroutines/qsvt.py

```diff
@@ -144,7 +144,8 @@ def qsvt(A, angles, wires, convention=None):
     projectors = projectors[::-1]  # reverse order to match equation
 
     if convention == "Wx":
```

Comment:
since 147's 'if global_phase' is only true in the case where 'convention=='Wx'', we can nix this line, s.t. the only req for passing prod(global_phase_op,QSVT(...)) is 'if global_phase'

Reply:
Good point! However, we cannot delete the line because then if the convention is not “wx” there is an error since it does not recognize `global_phase`

Reply:
lol of course, I'm seemingly trying to duplicate the initial error; looks good then